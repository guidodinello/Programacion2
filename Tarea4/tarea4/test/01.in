# Implementar structs, las operaciones de crear y liberar, cantidadEnIterador
cantidadEnIterador
# comando que reinicia las estructruras (no es reiniciarIterador)
reiniciar
agregarAIterador 3
cantidadEnIterador
reiniciar
agregarAIterador 3
agregarAIterador 3
cantidadEnIterador
reiniciar
agregarAIterador 3
agregarAIterador 3
agregarAIterador 3
agregarAIterador 3
agregarAIterador 3
agregarAIterador 3
agregarAIterador 3
agregarAIterador 3
cantidadEnIterador
Fin
