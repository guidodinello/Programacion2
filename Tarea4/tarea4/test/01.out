1>#  Implementar structs, las operaciones de crear y liberar, cantidadEnIterador.
2>0.
3>#  comando que reinicia las estructruras (no es reiniciarIterador).
4>Estructuras reiniciadas.
5>Agregando.
6>1.
7>Estructuras reiniciadas.
8>Agregando.
9>Agregando.
10>2.
11>Estructuras reiniciadas.
12>Agregando.
13>Agregando.
14>Agregando.
15>Agregando.
16>Agregando.
17>Agregando.
18>Agregando.
19>Agregando.
20>8.
21>Fin.
